libs/blocks/chart/chart.js

```diff
@@ -44,10 +44,34 @@ function processDataset(data) {
   return dataset;
 }
 
-function processSeries(data) {
-  const series = [];
-  // TODO: Series data
-  return series;
+const parseValue = (value) => (Number.isInteger(Number(value)) ? parseInt(value, 10) : value);
```

Comment:
I remember we discussed if this could instead be in a util. Curious if there was a stronger reason to just keep it here?